js/lib/ajax.js

```diff
@@ -102,6 +102,11 @@ elgg.ajax.handleOptions = function(url, options) {
  * @private
  */
 elgg.ajax.handleAjaxError = function(xhr, status, error) {
+	if (!xhr.getAllResponseHeaders()) {
```

Comment:
yep... but he did it the wrong way... with `status=abort` which did not work as expected.. i was working on that, but then found out about the token issue, so decided to fix all for 1.12